src/main/java/spim/headless/removeDetection/RemoveDetectionParameters.java

```diff
@@ -0,0 +1,187 @@
+import mpicbg.spim.data.sequence.ImgLoader;
+import mpicbg.spim.data.sequence.ViewDescription;
+import mpicbg.spim.data.sequence.ViewId;
+import simulation.imgloader.SimulatedBeadsImgLoader;
+import spim.fiji.spimdata.SpimData2;
+import spim.fiji.spimdata.boundingbox.BoundingBoxes;
+import spim.fiji.spimdata.interestpoints.InterestPoint;
+import spim.fiji.spimdata.interestpoints.ViewInterestPoints;
+import spim.headless.interestpointdetection.DoG;
+import spim.headless.interestpointdetection.DoGParameters;
+import spim.headless.interestpointdetection.InterestPointTools;
+import spim.process.removeDetection.DetectionRemoval;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+
+/**
+ * Created by Stephan Janosch on 01/07/15.
+ */
+public class RemoveDetectionParameters {
+
+    /**
+     * The view descriptions to work on
+     */
+    protected Collection< ViewDescription > toProcess;
+    /**
+     * The Label.
+     */
+    protected String label;
+    /**
+     * The Min.
+     */
+    protected double min;
+    /**
+     * The Max.
+     */
+    protected double max;
+    /**
+     * The Keep range.
+     */
+    protected boolean keepRange;
+
+
+    /**
+     * Gets to process.
+     *
+     * @return the to process
+     */
+    public Collection<ViewDescription> getToProcess() {
+        return toProcess;
+    }
+
+    /**
+     * Sets to process.
+     *
+     * @param toProcess the to process
+     */
+    public void setToProcess(Collection<ViewDescription> toProcess) {
+        this.toProcess = toProcess;
+    }
+
+    /**
+     * Gets label.
+     *
+     * @return the label
+     */
+    public String getLabel() {
+        return label;
+    }
+
+    /**
+     * Sets label.
+     *
+     * @param label the label
+     */
+    public void setLabel(String label) {
+        this.label = label;
+    }
+
+    /**
+     * Gets min.
+     *
+     * @return the min
+     */
+    public double getMin() {
+        return min;
+    }
+
+    /**
+     * Sets min.
+     *
+     * @param min the min
+     */
+    public void setMin(double min) {
+        this.min = min;
+    }
+
+    /**
+     * Gets max.
+     *
+     * @return the max
+     */
+    public double getMax() {
+        return max;
+    }
+
+    /**
+     * Sets max.
+     *
+     * @param max the max
+     */
+    public void setMax(double max) {
+        this.max = max;
+    }
+
+    /**
+     * Gets keep range.
+     *
+     * @return the keep range
+     */
+    public boolean getKeepRange() {
+        return keepRange;
+    }
+
+    /**
+     * Sets keep range.
+     *
+     * @param keepRange the keep range
+     */
+    public void setKeepRange(boolean keepRange) {
+        this.keepRange = keepRange;
+    }
+
+
+    /**
+     * A method which tests the functionality
+     *
+     * @param args the input arguments
+     */
+    public static void main(String[] args)
+    {
+
```

Comment:
Please simplify using the SpimData2.convert() method and the DoGParameters.testDoG()